tools/flang2/flang2exe/iltutil.cpp

```diff
@@ -331,6 +331,12 @@ dump_ilt(FILE *ff, int bihx)
     fprintf(ff, " UJRES");
   if (BIH_SIMD(bihx))
     fprintf(ff, " SIMD");
+  if (BIH_NOSIMD(bihx))
+    fprintf(ff, " NOSIMD");
```

Comment:
The patch fails here. These two lines (334-335) were already included in PR #660.